CDP4CrossViewEditor/ViewModels/ParameterTypeSelectorViewModel.cs

```diff
@@ -46,7 +46,7 @@ public sealed class ParameterTypeSelectorViewModel : ThingSelectorViewModel
         /// <summary>
         /// Hardcoded power related parameter type short names list
         /// </summary>
-        private static readonly string[] PowerParameters = { "Redundancy", "P_on", "P_stby", "P_peak", "P_duty_cyc", "P_mean" };
+        private static readonly string[] PowerParameters = { "redundancy", "P_on", "P_stby", "P_peak", "P_duty_cyc", "P_mean" };
```

Comment:
Additionally, the "add power parameter types" toggle does not add them in the proper order (see [this comment](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues/658#issuecomment-756666958)).

Reply:
In the `ExecuteMovePowerToTarget`/`ExecuteMovePowerToSource` methods:
```
var powerParameterTypes = this.ParameterTypeSourceList
    .Where(row => PowerParameters.Contains(row.Thing.ShortName));
```
needs to be replaced with
```
var powerParameterTypes = PowerParameters
    .Select(shortName => this.ParameterTypeSourceList.FirstOrDefault(row => string.Equals(row.Thing.ShortName, shortName, StringComparison.InvariantCultureIgnoreCase)))
    .Where(row => row != null);
```

Reply:
a) This was done on this PR #714
b) Changed.